pages/smart-account-modules.mdx

```diff
@@ -12,9 +12,9 @@ Safe Modules can include daily spending allowances, amounts that can be spent wi
 
 ## How to create a Safe Module
 
-A great way to understand how Safe Modules work is by creating one. An excellent place to start is [Safe Modding 101: Create your own Safe Module](https://www.youtube.com/watch?v=nmDYc9PlAic).
+A great way to understand how Safe Modules work is by creating one. An excellent place to start is [Safe Modding 101: Create your own Safe Module](https://www.youTube.com/watch?v=nmDYc9PlAic).
```

Comment:
I'm not sure this is the best solution. Links should be valid as they are

Reply:
Same here